vms/platformvm/state/state_test.go

```diff
@@ -1401,6 +1401,56 @@ func TestStateSubnetOwner(t *testing.T) {
 	require.Equal(owner2, owner)
 }
 
+func TestStateSubnetManager(t *testing.T) {
+	tests := []struct {
+		name  string
+		setup func(s State, subnetID ids.ID, chainID ids.ID, addr []byte)
+	}{
+		{
+			name: "in-memory",
+			setup: func(s State, subnetID ids.ID, chainID ids.ID, addr []byte) {
+				s.SetSubnetManager(subnetID, chainID, addr)
+			},
+		},
+		{
+			name: "cache",
+			setup: func(s State, subnetID ids.ID, chainID ids.ID, addr []byte) {
```

Comment:
Should this pass in the `*testing.T` or the `*require.Assertions`? Right now this function is executed by the inner test but uses the outer testing struct.